osbs/build/build_request.py

```diff
@@ -794,12 +802,6 @@ def render(self, validate=True):
             self.spec.validate()
         super(ProductionBuild, self).render()
 
-        repo_name = git_repo_humanish_part_from_uri(self.spec.git_uri.value)
-        # NOTE: Since only the repo name is used, a forked repos will have
-        # the same git-repo-name tag. This is a known limitation. If this
-        # use case must be handled properly, the git URI must be taken into
-        # account.
-        self.set_label('git-repo-name', repo_name)
         self.set_label('git-branch', self.spec.git_branch.value)
```

Comment:
Branch is only mandatory for production builds, though I can't remember why it's the case, maybe we can make it mandatory for all builds?